docs/docs/api/promise.coroutine.md

```diff
@@ -52,8 +54,6 @@ Running the example:
     Ping? 8
     ...
 
-When called, the coroutine function will start an instance of the generator and returns a promise for its final value.
```

Comment:
It is replaced by "When the returned function (e.g. the `ping` function in the example below) is called, it will start the generator (by calling its `next()`) and return immediately."